security/openssl-tests.py

```diff
@@ -58,6 +63,16 @@ def setUp(self):
             self.srcdir = smm.get_source(pkg_name, self.srcdir)
             if not self.srcdir:
                 self.fail("openssl source install failed.")
+        os.chdir(self.srcdir)
+        # Different Linux distro releases use different arguments
+        # for configure command. Provide an option to the user
+        # to specify the same.
```

Comment:
@sacsant if it is distro specific then can we hard code them like how we do for packages?

Reply:
Actually the comment is wrong. The configure parameters are tied to the openssl version and not necessarily distro version. For bistros using the default parameters to configure should work. In case a user wants to specify different/additional options they can do so using this newly introduced yaml parameter. This is particularly useful while using latest upstream code.

Thanks for the review comment. I have update the patch and reworded the above comment.